dynamodb-stream-reader/src/main/java/no/unit/nva/dynamodb/DynamoDBEventTransformer.java

```diff
@@ -43,18 +43,39 @@ public DynamoDBEventTransformer() {
      */
     public IndexDocument parseStreamRecord(DynamodbEvent.DynamodbStreamRecord streamRecord) {
         JsonNode record = toJsonNode(streamRecord);
+        IndexDate date = extractDate(record);
 
         return new IndexDocument.Builder()
                 .withType(extractType(record))
                 .withIdentifier(extractIdentifier(record))
                 .withContributors(extractContributors(record))
-                .withDate(extractDate(record))
+                .withDate(date)
                 .withTitle(extractTitle(record))
                 .build();
     }
 
-    private String extractDate(JsonNode record) {
-        return formatDate(extractYear(record), extractMonth(record), extractDay(record));
+    private IndexDate extractDate(JsonNode record) {
```

Comment:
Could extractDate go into the IndexDate class? This would mean that either the related static constants should also be moved either to the IndexedDate class, or all the constants to be moved in their own class